index.js

```diff
@@ -3,8 +3,8 @@ export function includeKeys(object, predicate) {
 
 	if (Array.isArray(predicate)) {
 		const set = new Set(predicate);
-		for (const key of Object.keys(object)) {
-			if (set.has(key)) {
+		for (const key of Reflect.ownKeys(object)) {
+			if (isEnumerable.call(object, key) && set.has(key)) {
```

Comment:
Unlike `Object.keys()`, `Reflect.ownKeys()` also returns symbols. However, it also returns non-enumerable properties, so we need an additional check.